### DISCUSSION TITLE
Is there any support for Spans?

### DISCUSSION THREAD

**Discussion by AkshatDogra** [tecoholic/ner-annotator#99] [Q&A]
Say I have something like Microsoft Office. I want to add Microsoft Office to the Technology tag and Microsoft to the Company tag. Can I do this with ner-annotator?

**Comment by alvi-khan** [comment]
Unfortunately we do not support nested annotations yet. The issue is being tracked [here](https://github.com/tecoholic/ner-annotator/issues/86).

**Comment by Nandu3219** [comment]
how to annotate multiple files

**Reply by alvi-khan** [reply]
Hello @Nandu3219! Once you're done annotating your first file, you can export the annotations and then open a new file to start annotating that one. These options are in the menu bar on the top-left of the app (shown in the screenshots).

![image](https://github.com/tecoholic/ner-annotator/assets/55395950/57aec36c-9786-4c0e-88cf-6cdc7f426b6c)
![image](https://github.com/tecoholic/ner-annotator/assets/55395950/2f1782dd-5804-44a4-ab8b-4abd0f01bda7)

For future reference, please try to ask questions in a related discussion if there is one or create a new discussion if required. Having unrelated questions in the same discussion will quickly make things confusing for everyone.

**Reply by Nandu3219** [reply]
thanks for your reply.....

but I am using windows application which you provided
there is no option for file , may be i am using any old application or what 
if i want to annotate another file i need to restart/reopen the file

**Reply by alvi-khan** [reply]
I believe you are using an older version of the application. Could you please confirm the version number? You should be able to find it under `Help -> About`.

The latest version can be downloaded from [here](https://github.com/tecoholic/ner-annotator/releases/latest). There shouldn't be any differences between the web version and the Windows version in terms of UI or functionality.

**Reply by Nandu3219** [reply]
Thank you ...!

yes previous version is 1.0.3

now I am able to see file option and also with dark mode option 
and to select taga no need to scroll top 

thanks for quick reply ..... :)
